x/ibc/core/23-commitment/types/merkle.go

```diff
@@ -270,8 +270,11 @@ func verifyChainedMembershipProof(root []byte, specs []*ics23.ProofSpec, proofs
 }
 
 // Empty returns true if the root is empty
```

Comment:
Thanks for fixing this!

I think this godoc should be moved to the Empty func and maybe add a comment that the blank vars are for performance?

Reply:
Done, thank you @colin-axner!